scripts/setup-pepper-capture.sh

```
@@ -0,0 +1,469 @@
+
+  log "Launching emulator '${AVD_NAME}' with writable system..."
+  emulator -avd "${AVD_NAME}" -writable-system -gpu host -no-snapshot-load &
+  local EMU_PID=$!
+
+  wait_for_device
+
+  log "Emulator running (PID: ${EMU_PID})"
+  log "Installing Pepper APK..."
+
+  if [ -f "${APK_FILE}" ]; then
+    adb install -r "${APK_FILE}" && log "Pepper installed" || warn "APK install failed — may need different version"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)